navigation/build.gradle

```diff
@@ -0,0 +1,66 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.2"
+
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles 'consumer-rules.pro'
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
```

Comment:
Not sure how it works with library dependencies, will R8 be enabled if this isn't true, but the app which is using the dependency has it enabled? If not, then I think we need to set this to true. (And provide consumer rules if needed, but it's not the case I think.)

Reply:
https://developer.android.com/studio/projects/android-library
As far as I understand, proguard/r8 runs on the actuall app module, at which point it will only consider the proguard config file from the aar.